contribs/gmf/less/mobilelayertree.less

```diff
@@ -21,4 +21,8 @@ nav.nav-left .gmf-layertree-node a[data-toggle] {
   .group {
     font-size: @font-size-base;
   }
+
+  [ngeo-popover] {
+    display: none;
+  }
```

Comment:
Sorry to be picky but if this it's not related to current pull request, then it should be addressed in a different pull request or at least in a separate commit.